### PR TITLE
add projects by nikgraf

### PR DESCRIPTION
**Pull request by nikgraf**

### CODE REVIEW

**Comment by nikgraf** [comment]
@IonicaBizau how to add projects done by multiple Austrians together (simply add two authors?) or is this only for personal projects?

**Comment by IonicaBizau** [comment]
@nikgraf Maybe we can make a first exception here. Just add the repository in the list. :+1:

**Comment by nikgraf** [comment]
@IonicaBizau nevermind, we need to do some maintenance anyway first 😄 

btw it was this one https://github.com/carteb/carte-blanche together with @mxstbr

File: package.json [hunk 1]
```diff
@@ -98,6 +98,18 @@
               ""
             ],
             [
+              "1597",
+              "[@nikgraf](https://github.com/nikgraf)/[**github**](https://github.com/nikgraf/belle)",
```
Comment (pa7) [inline package.json]: that will result in [@nikgraf](https://github.com/nikgraf)/[**github**](https://github.com/nikgraf/belle) , you might want to rename it to [@nikgraf](https://github.com/nikgraf)/[**belle**](https://github.com/nikgraf/belle) :)

Comment (IonicaBizau) [reply package.json]: @nikgraf Yes, can you please fix that?

Comment (nikgraf) [reply package.json]: @IonicaBizau just did

**Comment by IonicaBizau** [comment]
@nikgraf Cool! Sorry for delay. 🚀

**Pull request merged**